packages/client/test/unit/Write.test.ts

```diff
@@ -0,0 +1,285 @@
+import {expect} from 'chai'
+import nock from 'nock' // WARN: nock must be imported before NodeHttpTransport, since it modifies node's http
+import {
+  ClientOptions,
+  HttpError,
+  WriteOptions,
+  Point,
+  InfluxDBClient,
+  WritePrecision,
+} from '../../src'
+import {collectLogging, CollectedLogs, unhandledRejections} from '../util'
+import zlib from 'zlib'
+import {rejects} from 'assert'
+import {isDefined} from '../../src/util/common'
+
+const clientOptions: ClientOptions = {
+  url: 'http://fake:8086',
+  token: 'a',
+}
+const BUCKET = 'bucket'
```

Comment:
Rename to `database`:

```suggestion
const DATABASE = 'database'
```